gitwrapperlib/gitwrapperlib.py

```diff
@@ -159,6 +165,25 @@ def get_current_branch(self):
                      if branch.startswith('*')),
                     None)
 
+    def get_default_branch(self):
+        """Returns the remote default branch."""
+        branch = None
+
+        try:
+            remote = str(self._git.remote('show', 'origin'))
+            match = re.search(r'HEAD branch: (\S+)', remote)
+            if match:
+                branch = match.group(1)
+                if branch == '(unknown)':
+                    branch = None
+        except sh.ErrorReturnCode:
+            raise RemoteNotFound
+
+        if not branch:
+            raise BranchNotFound
```

Comment:
Maybe a little help text here elaborating? Like remote is set to (unknown) please check your remote settings?